src/utils/computePercentage.test.ts

```diff
@@ -21,6 +21,24 @@ test("Return 0 if progressBarRef.current is undefined", () => {
   ).toBe(0);
 });
 
+test("Return 0 if volumeBarRef.current is undefined", () => {
+  expect(
+    computePercentageOfY(new MouseEvent("mousedown"), {
+      current: null,
+    })
+  ).toBe(0);
+  expect(
+    computePercentageOfY(new MouseEvent("mousedown"), {
+      current: null,
+    })
+  ).toBe(0);
+  expect(
+    computePercentageOfY(new MouseEvent("mousedown"), {
+      current: null,
+    })
+  ).toBe(0);
+});
+
```

Comment:
As there's more than one functions being test here, please group their tests accordingly using the [`describe` API](https://vitest.dev/api/#describe), so that the tests look clearer.